scripts/issue/1139/acl_reorg.py

```diff
@@ -0,0 +1,198 @@
+from brownie import accounts, chain, Contract, interface, multicall, web3
+
+from great_ape_safe import GreatApeSafe
+from helpers.addresses import r
+from rich.console import Console
+
+C = Console()
+
+dev_msig = GreatApeSafe(r.badger_wallets.dev_multisig)
+techops = GreatApeSafe(r.badger_wallets.techops_multisig)
+
+# General role
+DEFAULT_ADMIN_ROLE = (
+    "0x0000000000000000000000000000000000000000000000000000000000000000"
+)
+# BadgerTree
+PAUSER_ROLE = web3.solidityKeccak(["string"], ["PAUSER_ROLE"]).hex()
+UNPAUSER_ROLE = web3.solidityKeccak(["string"], ["UNPAUSER_ROLE"]).hex()
+ROOT_PROPOSER_ROLE = web3.solidityKeccak(["string"], ["ROOT_PROPOSER_ROLE"]).hex()
+ROOT_VALIDATOR_ROLE = web3.solidityKeccak(["string"], ["ROOT_VALIDATOR_ROLE"]).hex()
+# Rewards Logger
+MANAGER_ROLE = web3.solidityKeccak(["string"], ["MANAGER_ROLE"]).hex()
+# Guardian
+APPROVED_ACCOUNT_ROLE = web3.solidityKeccak(["string"], ["APPROVED_ACCOUNT_ROLE"]).hex()
+# Keeper ACL
+EARNER_ROLE = web3.solidityKeccak(["string"], ["EARNER_ROLE"]).hex()
+HARVESTER_ROLE = web3.solidityKeccak(["string"], ["HARVESTER_ROLE"]).hex()
+TENDER_ROLE = web3.solidityKeccak(["string"], ["TENDER_ROLE"]).hex()
+# registryAccessControl
+DEVELOPER_ROLE = web3.solidityKeccak(["string"], ["DEVELOPER_ROLE"]).hex()
+
+# techops signers
+techops_signers = list(
+    interface.IGnosisSafe_v1_3_0(r.badger_wallets.techops_multisig).getOwners()
+)
+# print them in terminal for visibilitys
+C.print(techops_signers)
+
+GUARDIAN_BACKUPS = [
+    r.badger_wallets.guardian_backups.ops_executor7,
+    r.badger_wallets.guardian_backups.defender1,
+    r.badger_wallets.guardian_backups.defender2,
+]
+DEVELOPER_ROLE_MEMBERS = list(techops_signers + GUARDIAN_BACKUPS)
```

Comment:
Did you mean `APPROVED_ACCOUNT_ROLE_MEMBERS`?

Reply:
you're right, i renamed it on dc7554ef6fcfcd4614b4936bd7dbdba67573ebd9